src/routes/(site)/s23positions/data.ts

```diff
@@ -0,0 +1,93 @@
+import type { ListItem } from '$lib/components/recursive-ul/types';
+import { TEAMS } from '$lib/public/board/data';
```

Comment:
'TEAMS' is defined but never used.

Reply:
Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/commit/3ef17687c40e597e980d61db5f9d67035076c3e4>.